client/src/components/MarketUpdate/TurnipFormSelectors.js

```diff
@@ -0,0 +1,31 @@
+export function getDays() {
+	const now = new Date();
+
+	const times = {
+		sunday: new Date(),
+		monday: new Date(),
+		tuesday: new Date(),
+		wednesday: new Date(),
+		thursday: new Date(),
+		friday: new Date(),
+		saturday: new Date(),
+	};
+
+	//0 Sunday
```

Comment:
Comment doesn't really mean much. I would remove. 

Additional info would help here, though. Maybe something like: 

Build option list of weekday choices from Sunday to Saturday for `HTMLSelectElement` on MarketUpdate page